docs/usage-guide/topics/ch11-resumption.md

```diff
@@ -25,6 +25,8 @@ Servers should set the three caching callback functions: `s2n_config_set_cache_s
 
 Clients should call `s2n_connection_get_session()` to retrieve some serialized state about the session. Then `s2n_connection_set_session()` should be called with that saved state when attempting to resume a new connection.
 
+Applications must perform a graceful TLS shutdown to be able to resume a session with session caching. Any errors during the connection will result in that specific session being removed from the cache, making it unavailable for resumption. This includes an abrupt connection termination where no `close_notify` alert is received by the server. For more information on how to properly close connections, see [Closing the Connection](./ch07-io.md#closing-the-connection)
```

Comment:
Yeah this sounds good. I wonder if your first sentence fits better towards the end though?
```suggestion
Any errors during the connection will result in that specific session being removed from the cache, making it unavailable for resumption. This includes an abrupt connection termination where no `close_notify` alert is received by the server. Applications must perform a graceful TLS shutdown to be able to resume a session with session caching. For more information on how to properly close connections, see [Closing the Connection](./ch07-io.md#closing-the-connection)
```